### PR TITLE
NEPT-1889: Integration of Registry on steroids contrib module.

### PR DESCRIPTION
**Pull request by drupol**
### Description

Upgrade to Atomium 7.x-3.x
It requires [Alternatives](https://www.drupal.org/project/alternatives) and [Registry on steroids](https://www.drupal.org/project/registryonsteroids) contrib modules.

### Commands

drush en registryonsteroids -y
drush en registryonsteroids_alter -y
drush cc all

### CODE REVIEW

**Comment by jfhovinne** [comment]
To rebase against release-2.4, see NEPT-1832.

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -606,6 +606,11 @@ projects[registration][version] = "1.6"
 projects[registry_autoload][subdir] = "contrib"
 projects[registry_autoload][version] = 1.3
 
+projects[registryonsteroids][type] = module
+projects[registryonsteroids][download][type] = git
+projects[registryonsteroids][download][url] = https://github.com/drupol/registryonsteroids.git
+projects[registryonsteroids][download][branch] = 7.x-1.x
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: Can we add this dependency with d.o?
```
 projects[registryonsteroids][subdir] = "contrib"
 projects[registryonsteroids][version] = 1.0
```

File: resources/multisite_drupal_standard.make [hunk 2]
```diff
@@ -827,6 +832,9 @@ projects[wysiwyg][download][version] = "2.4"
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-9874
 projects[wysiwyg][patch][] = patches/wysiwyg-js-url-9874.patch
 
+projects[xautoload][subdir] = "contrib"
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: We need to add this module for the list of module reviews by the QA.

File: resources/multisite_drupal_standard.make [hunk 3]
```diff
@@ -1034,7 +1042,9 @@ projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
 projects[ec_resp][download][tag] = 2.3.6
 
 projects[atomium][type] = theme
-projects[atomium][version] = 2.7
+projects[atomium][download][type] = git
+projects[atomium][download][url] = https://github.com/ec-europa/atomium.git
+projects[atomium][download][branch] = nept-1889
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: Do not forget to change this part with:
`projects[atomium][version] = 3.0`

**Review by voidtek — CHANGES_REQUESTED** [review]

**Comment by voidtek** [comment]
Is it not necessary to include a hook_update to enabling these modules? and remove the manual commands for devops, because we need to enable them for all sub-sites?

File: resources/multisite_drupal_standard.make [hunk 4]
```diff
@@ -1034,7 +1042,9 @@ projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
...
 
 projects[ec_europa][type] = theme
```
Comment (voidtek) [inline resources/multisite_drupal_standard.make:1141]: Don't forget to update the version of ec_europa.

**Review by voidtek — CHANGES_REQUESTED** [review]

File: resources/multisite_drupal_standard.make (hunk 3 above)
Comment (drupol) [reply resources/multisite_drupal_standard.make]: It's not yet ready, this is why I'm using the branch, but I will as soon as Atomium 3.0 is ready.

**Comment by drupol** [comment]
@voidtek You added xautoload but we are going to use registry_autoload. This is why we use the module Alternatives.

**Comment by richardcanoe** [comment]
Tested in conjunction with [ec_europa](https://github.com/ec-europa/atomium/pull/105) and [atomium](https://github.com/ec-europa/ec_europa/pull/64).

Works well locally.

**Pull request closed without merging**